reducers/storage.ts

```diff
@@ -110,6 +111,47 @@ function storage(state: Record<string, any> = {}, action: GenericAction) {
     }
 }
 
+function migrateDrafts(state: any) {
+    const drafts: any = {};
+    for (const storageKey in state) {
```

Comment:
This should be `for (const storageKey of Object.keys(state))`. We generally avoid using for..in because it has some odd behaviour.

Alternatively, you could use `for (const [storageKey, storageDraft] of Object.entries(state))` to get both those at once